File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraTreeList;

namespace WindowsApplication1 {
	public partial class Form1 : Form {
		public Form1() {
			InitializeComponent();
		}

        private void treeList1_CustomDrawNodeCell(object sender, CustomDrawNodeCellEventArgs e) {
            TreeList tl = sender as TreeList;
            if (e.Node == tl.FocusedNode) {
                e.Graphics.FillRectangle(SystemBrushes.Window, e.Bounds);
                Size textSize = e.Appearance.CalcTextSize(e.Cache, e.CellText, e.Bounds.Width).ToSize();
                Rectangle rect = new Rectangle(e.EditViewInfo.ContentRect.Left,
                    e.EditViewInfo.ContentRect.Top, textSize.Width, textSize.Height);
                e.Cache.FillRectangle(SystemBrushes.Highlight, rect);
                e.Cache.DrawString(e.CellText, e.Appearance.Font, SystemBrushes.HighlightText, rect, e.Appearance.GetStringFormat());
                e.Handled = true;
            }
        }

		private void Form1_Load(object sender, EventArgs e) {
			treeList1.OptionsBehavior.Editable = false;
			treeList1.OptionsView.ShowColumns = false;
			treeList1.OptionsView.ShowIndicator = false;
			treeList1.OptionsView.ShowHorzLines = false;
			treeList1.OptionsView.ShowVertLines = false;
			
		}
	}
}

File: VB/WindowsApplication1/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraTreeList

Namespace WindowsApplication1
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub treeList1_CustomDrawNodeCell(ByVal sender As Object, ByVal e As CustomDrawNodeCellEventArgs) Handles treeList1.CustomDrawNodeCell
            Dim tl As TreeList = TryCast(sender, TreeList)
            If e.Node.Equals(tl.FocusedNode) Then
                e.Graphics.FillRectangle(SystemBrushes.Window, e.Bounds)
                Dim textSize As Size = e.Appearance.CalcTextSize(e.Cache, e.CellText, e.Bounds.Width).ToSize()
                Dim rect As New Rectangle(e.EditViewInfo.ContentRect.Left, e.EditViewInfo.ContentRect.Top, textSize.Width, textSize.Height)
                e.Cache.FillRectangle(SystemBrushes.Highlight, rect)
                e.Cache.DrawString(e.CellText, e.Appearance.Font, SystemBrushes.HighlightText, rect, e.Appearance.GetStringFormat())
                e.Handled = True
            End If
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
            treeList1.OptionsBehavior.Editable = False
            treeList1.OptionsView.ShowColumns = False
            treeList1.OptionsView.ShowIndicator = False
            treeList1.OptionsView.ShowHorzLines = False
            treeList1.OptionsView.ShowVertLines = False

        End Sub
    End Class
End Namespace